definitions/variable/energy/energy-secondary.yaml

```diff
@@ -34,6 +34,18 @@
     description: electricity losses from long-range high-voltage transmission
     unit: [EJ/yr, GWh]
 
+#ONSSET START
+- Secondary Energy|Electricity|Transmission Losses|Share:
+    description: This value represents the country’s average technical losses on transmission and distribution
+    unit: %
+- Secondary Energy|Electricity|Distance|Maximum:
```

Comment:
Do you change these values dynamically per year and is this part of the model output of the ONSSET model? 
To me this looks more like model parameters in which case they are typically not included in the result files.